ipi/engine/forcefields.py

```diff
@@ -497,42 +511,56 @@ def poll(self):
                 if r["status"] == "Queued":
                     r["status"] = "Running"
                     r["t_dispatched"] = time.time()
-                    self.evaluate(r) # MR BAD
+                    self.evaluate(r)  # MR BAD
 
     def evaluate(self, r):
-        """Just a silly function evaluating a non-cutoffed, non-pbc and
-        non-neighbour list dmd."""
+        """Evaluating dmd: pbc: YES,
+        cutoff: NO, neighbour list: NO."""
```

Comment:
You OK having this as the first version or you want to get more features in?

Reply:
No, I don't think this needs any new features, but still it is good to call attention to what is there and what is not. It uses just minimal im. conv.